### PR TITLE
feat:david/service fee on rent payment

### PR DESCRIPTION
**Pull request by Md-coder**
Update wallet.ts

### CODE REVIEW

File: src/interfaces/wallet.ts [hunk 1]
```diff
@@ -15,12 +15,11 @@ export interface ITransaction {
   balance: number;
   timestamp: number;
   notes: string;
-  trxLogId: string;
+  trxLogId?: string;
```
Comment (nomadyus) [inline src/interfaces/wallet.ts]: This is not ideal. We always want to have a log entry for all wallet transactions.

**Review by nomadyus — CHANGES_REQUESTED** [review]

File: src/interfaces/wallet.ts (hunk 1 above)
Comment (nomadyus) [reply src/interfaces/wallet.ts]: Addressing the credit structure comment in `realhaus` repo will likely cause this change to be reverted.

Comment (dijistanley) [reply src/interfaces/wallet.ts]: On another thought, @nomadyus is right. please revert this code @Md-coder

**Review by dijistanley — CHANGES_REQUESTED** [review]
Please discard this PR

**Pull request closed without merging**